### ISSUE TITLE
[Failing Test] up failing in master-upgrade

### ISSUE THREAD

**Issue by alejandrox1** [kubernetes/kubernetes#76895]
**Which jobs are failing**:
* `ci-kubernetes-e2e-gce-master-new-downgrade-cluster`
* `ci-kubernetes-e2e-gce-master-new-downgrade-cluster-parallel`

**Which test(s) are failing**:
* `Up` ([spyglass](https://prow.k8s.io/view/gcs/kubernetes-jenkins/logs/ci-kubernetes-e2e-gce-master-new-downgrade-cluster/1120304086290272258))

**Since when has it been failing**:
4/16. 
It is also interesting to notice that there are about 6 runs before 4/16 that are still marked as running.

**Testgrid link**:
* https://testgrid.k8s.io/sig-release-master-upgrade#gce-master-new-downgrade-cluster
* https://testgrid.k8s.io/sig-release-master-upgrade#gce-master-new-downgrade-cluster-parallel

**Reason for failure**:
The error message is:
```
error during ./hack/e2e-internal/e2e-up.sh: exit status 2
```

This also showed up in the logs:
```
Cluster failed to initialize within 300 seconds.
```

/kind failing-test
/priority important-soon
/sig testing
/milestone v1.15

/cc @jimangel @smourapina @rarchk @alenkacz

**Comment by imkin** [comment]
/sig gcp

This jobs seems to have been failing from quite some time (even before 4/16)
https://prow.k8s.io/job-history/kubernetes-jenkins/logs/ci-kubernetes-e2e-gce-master-new-downgrade-cluster

@Katharine pointed out that the jobs failures before 2019-04-16 and after 2019-04-27 09:28:04 +0000 UTC  are similar. 

This corresponds to the commit https://github.com/kubernetes/kubernetes/pull/74690 and revert https://github.com/kubernetes/kubernetes/pull/77158

In other words after the revert https://github.com/kubernetes/kubernetes/pull/77158 we are no longer failing the `Up`  tests.

I think sig-gcp should take a look at this. 

One concern I see is that the configuration of these downgrade tests attempts to use VERY old unsupported version of etcd for some reason.  https://prow.k8s.io/rerun?prowjob=12cc9b85-69c7-11e9-9dd9-0a580a6c084f

**Comment by wenjiaswe** [comment]
https://github.com/kubernetes/kubernetes/pull/74690 enables mtls between etcd server and kube-apiserver on master vm, so I can imagine it would have impact on gcp master upgrade test. I know kubemark test is changed accordingly https://github.com/kubernetes/kubernetes/pull/76017.

**Comment by alejandrox1** [comment]
#9679 may be of interest here.
Thank you @imkin @Katharine @wenjiaswe for taking a look at this

**Comment by imkin** [comment]
@wenjiaswe since #74690 is reverted there is no issue with it. 

The failure has changed now. I see the failure messages in the api-server as is not able to start

```
unexpected ListAndWatch error: storage/cacher.go:/priorityclasses: Failed to list *scheduling.PriorityClass: no kind "PriorityClass" is registered for version "scheduling.k8s.io/v1" in scheme "k8s.io/kubernetes/pkg/api/legacyscheme/scheme.go:29"
```
https://storage.googleapis.com/kubernetes-jenkins/logs/ci-kubernetes-e2e-gce-master-new-downgrade-cluster/1123164548979757056/artifacts/bootstrap-e2e-master/kube-apiserver.log

**Comment by imkin** [comment]
@alejandrox1 I think you have linked the wrong PR.

**Comment by wenjiaswe** [comment]
@imkin Thanks for letting me know, I will probably add that in later when the performance regression investigation are done, but I know what went wrong with that PR and I will deliver with the correction.

**Comment by alejandrox1** [comment]
yes, I did, sorry... I meant to link to https://github.com/kubernetes/test-infra/issues/9679

**Comment by alejandrox1** [comment]
no test runs are showing up in testgrid since 4/27
this may related to https://github.com/kubernetes/test-infra/issues/12449

**Comment by alejandrox1** [comment]
@wenjiaswe do you have any updates?

**Comment by wenjiaswe** [comment]
@alejandrox1 Sorry I didn't follow up since  #74690 was reverted in #77158 and it's not the issue now. 

cc @jennybuckley who is helping to take look at gcp test failures.

**Comment by alejandrox1** [comment]
The tests mentioned in this issue can now be found in 
* https://testgrid.k8s.io/sig-release-master-informing#gce-master-new-downgrade-cluster
* https://testgrid.k8s.io/sig-release-master-informing#gce-master-new-downgrade-cluster-parallel

**Comment by alejandrox1** [comment]
@wenjiaswe Thank you very much for the update.
I will close this issue as the problem  with these tests seems to be about timeout i.e., https://prow.k8s.io/view/gcs/kubernetes-jenkins/logs/ci-kubernetes-e2e-gce-master-new-downgrade-cluster/1126813522928340995
/close
